gcb/makestage/cloudbuild.yaml

```diff
@@ -33,7 +41,9 @@ substitutions:
   _CM_REPO: "https://github.com/cert-manager/cert-manager.git"
   _KMS_KEY: "projects/cert-manager-release/locations/europe-west1/keyRings/cert-manager-release/cryptoKeys/cert-manager-release-signing-key/cryptoKeyVersions/1"
   _RELEASE_TARGET_BUCKET: "cert-manager-release"
-  _BUILDER_IMAGE_TAG: "20220629-ee75d11-4.2.1"
+  _BUILDER_IMAGE_TAG: "20240422-6b43e85-bookworm"
 
 options:
-  machineType: n1-highcpu-32
+  # https://cloud.google.com/build/docs/optimize-builds/increase-vcpu-for-builds
+  # https://cloud.google.com/build/pricing
+  machineType: E2_HIGHCPU_32
```

Comment:
N1_HIGHCPU_32 is still supported, but the official list of machine types suggests E2_HIGHCPU_32 instead.